src/eq_schema/block-types/listCollector/drivingQuestion.js

```diff
@@ -54,7 +54,7 @@ class DrivingQuestion {
     const list = getList(ctx, page.listId);
     this.answers = [
       {
-        id: `answer-driving-${pages[pages.length - 1].id}`,
+        id: `answer-driving-${page.answers[0].id}`,
```

Comment:
This no longer needs to be prefixed with 'answer-driving', it can be just 'answer' like all other answers.